21_node_management/22_search_nodes.txt

```diff
@@ -6,177 +6,116 @@
-
-.is: keywords
-[frame="topbot",options="header"]
-|======================
-|keyword |Description
-|node      |Nodes
-|group     |Groups
-|parameter |Parameters
-|directive |Directives
-|rule      |Rules
-|======================
+==== General behavior
 
-.in: keywords (common)
-[frame="topbot",options="header"]
-|======================
-|keyword |Search for
-|name |Names
-|id |IDs
-|description, long_description |Descriptions
-|enabled |Enabled elements (true or false)
-|======================
+In the Advanced Search tool, you can create complex searches based on Node
+Inventory information. The benefit of the Advanced Search tool is to save the
+query and create a Group of Nodes based on the search criteria.
 
-.in: keywords (nodes)
-[frame="topbot",options="header"]
-|======================
-|keyword |Search for
-|hostname |Hostnames
-|os_type |OS types (windows, linux, aix...)
-|os_name |OS Names (Windows Server 2012, Debian... )
-|os_version |OS versions (8, 2008 R2, ...)
-|os |OS Full Names (Debian GNU/Linux 6.0.10 (squeeze)...)
-|os_kernel_version |OS Kernel versions (3.16, 5.1...)
-|os_service_pack |OS Service Packs (for Windows and SuSE Linux)
-|architecture |OS Architectures (amd64, x86_64, i386)
-|ram |Machine memory
-|ips |Network IP addresses
-|policy_server_id |ID of a node's policy server (root...)
-|properties |Node properties (arbitrary key=values associated to a node)
-|rudder_roles |Rudder roles (rudder-reports, rudder-ldap...)
-|======================
+- 1. Select a field
 
-.in: keywords (groups)
-[frame="topbot",options="header"]
-|======================
-|keyword |Search for
-|dynamic |Dynamic groups
-|======================
+The selection of the field upon which the criteria will apply is a two step
+process. The list of fields is not displayed unordered and extensively. Fields
+have been grouped in the same way they are displayed when you look at
+information about a Node. First you choose among these groups: 'Node', 'Network
+Interface', 'Filesystem', 'Machine', 'RAM', 'Storage', 'BIOS', 'Controller',
+'Port', 'Processor', 'Sound Card', 'Video Card', 'Software', 'Environment
+ Variable', 'Processes', 'Virtual Machines'; then you choose
+among the list of fields concerning this theme.
 
-.in: keywords (directives)
-[frame="topbot",options="header"]
-|======================
-|keyword |Search for
-|dir_param_name |Directive parameter names, as in the Techniques metadata.xml ("GENERIC_FILE_CONTENT_PATH"...)
-|dir_param_value |Directive parameter values
-|technique_id |Technique IDs
-|technique_name |Technique names ("Enforce a file content"...)
-|technique_version |Technique version
-|======================
+- 2. Select the matching rule
 
-.in: keywords (parameters)
-[frame="topbot",options="header"]
-|======================
-|keyword |Search for
-|parameter_name |Parameter names
-|parameter_value |Parameter values
-|======================
+// FIXME: due to a bug in asciidoc, we cannot write '≠' 
```

Comment:
is it still true ?

Reply:
no idea :) I test and we will see if it breaks